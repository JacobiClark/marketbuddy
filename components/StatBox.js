import React from "react";

import { Box, Text } from "@chakra-ui/react";

const StatBox = ({ statType, statValue }) => {
  return (
    <Box
      p="4"
      width={{
        base: "50%", // 0-48em
        md: "25%", // 48em-80em,
        xl: "20%", // 80em+
      }}
    >
      <Text fontWeight="bold">{statType}</Text>
      <Text>{statValue}</Text>
    </Box>
  );
};

export default StatBox;
