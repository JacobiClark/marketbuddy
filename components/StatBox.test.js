import React from "react";
import renderer from "react-test-renderer";

import StatBox from "./StatBox";

it("Renders correctly without items", () => {
  const tree = renderer.create(<StatBox />).toJSON();
  expect(tree).toMatchSnapshot();
});
