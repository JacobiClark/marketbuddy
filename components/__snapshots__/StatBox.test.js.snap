// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly without items 1`] = `
Array [
  <style
    dangerouslySetInnerHTML={
      Object {
        "__html": ".css-17jybg2{padding:4px;}.css-17jybg2 width{base:50%;md:25%;xl:20%;}",
      }
    }
    data-emotion="css 17jybg2"
  />,
  <div
    className="css-17jybg2"
  >
    <style
      dangerouslySetInnerHTML={
        Object {
          "__html": ".css-in3yi3{font-weight:bold;}",
        }
      }
      data-emotion="css in3yi3"
    />
    <p
      className="chakra-text css-in3yi3"
    />
    <p
      className="chakra-text css-0"
    />
  </div>,
]
`;
